src/searcher/kiwix-search.cpp

```diff
@@ -104,7 +104,14 @@ int main(int argc, char** argv)
 
   if (reader) {
     searcher = new kiwix::Searcher();
-    searcher->add_reader(reader);
+    bool contians_FTIndex=searcher->add_reader(reader);
+    if(!contians_FTIndex){
+    	std::cerr<<"The Zim file does not contain a full-text index"<<endl;
```

Comment:
- Do not mix spaces with tab.
- Please add space around the `<<`.
- Add a `.` at the end of the sentence.
- It should be `std::endl`.